dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/dispatcher/internal/channel/HttpDispatcherLink.java

```diff
@@ -462,10 +487,21 @@ private void wrapHandlerAndExecute(Runnable handler) {
                 taskWrapper.setClassifiedExecutor(classifyExecutor);
                 classifyExecutor.execute(taskWrapper);
```

Comment:
```suggestion
                classifyExecutor.execute(handlerWC != null ? handlerWC : taskWrapper);
```
Or because it is the classifyExecutor it doesn't need to call the work context wrapped runnable?